docs/kcl-std/functions/std-array-map.md

```diff
@@ -14,19 +14,19 @@ map(
 ): [any]
 ```
 
-Given a list like `[a, b, c]`, and a function like `f`, returns
-`[f(a), f(b), f(c)]`
+Given a list like [`a, b, c`], and a function like `f`, returns
```

Comment:
I'm guessing that the brackets used to be inside the backticks because brackets are markdown syntax for links. If we want a literal `[`, we should use `\[`.